paper/Astic_Oldenburg_PGI_article_revised_v2.tex

```diff
@@ -0,0 +1,923 @@
+
+Lastly, we comment upon how important the development of open-source software packages has been for our research. It has allowed us to benefit from the work carried out by others. The development of the MAP-EM algorithm has been facilitated by the MLE code available through the open source Scikit-Learn project, a library of tools for machine-learning \citep{scikit-learn}. The successful application of our methodology to various geophysical survey methods, such as potential fields, DC, MT or FDEM, has been made possible because of the interconnectivity of SimPEG, an open source package to carry out geophysical inversions \citep{Cockett2015,heagy2017framework,SimPEGEM1DPoster}. To ensure reproducibility of the results in this paper, and to contribute to the development of the open source community, we are making the examples presented in this paper available online \citep{PGIexamples}, through the use of Jupyter Notebooks \citep{Perez2015}. This allows researchers to readily use our framework and also to contribute to the development of inversion codes that incorporate geophysical,  geological and petrophysical data to yield meaningful solutions.
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section{Acknowledgments}
+
+We thank Tim Munday (CSIRO) for making the Bookpurnong datasets available. We also thank all of the developers who have contributed to the SimPEG package (\url{https://simpeg.xyz}). In particular we thank Lindsey J. Heagy for her development of architectural features that allowed us to use flexible regularization functions and also to connect forward modellings that were based upon different physics, and Seogi Kang and Dominique Fournier for their implementation of the parallelized stitched EM1D code in SimPEG as well as for the many fruitful conversations. A special thanks to Rowan Cockett as a main founder of the SimPEG project, and to the entire SimPEG group for their support and for many fruitful conversations.
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\bibliographystyle{gji}
+\bibliography{bibliography}
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% \appendix
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\appendix{From A Posterior with Gaussian Distributions to an least-squares Objective-Function} \label{appendix:PDtoL2}
```

Comment:
👀 I think the commented-out part is is ok, there's just more up here 😄

Reply:
ah, thank you :)  late evening commits are dangerous! ;)

Reply:
Haha, oh yeah, I'm also guilty of the dangerous late night commits...